src/test/java/games/strategy/engine/framework/startup/login/ClientLoginValidatorTests.java

```diff
@@ -27,23 +26,23 @@
   private static final String PASSWORD = "password";
   private static final String OTHER_PASSWORD = "otherPassword";
 
-  private static final String MAGIC_MAC_START = ClientLoginValidator.MAC_MAGIC_STRING_PREFIX;
-
   public static final class MacValidationTest {
+    private static final String MAGIC_MAC_START = ClientLoginValidator.MAC_MAGIC_STRING_PREFIX;
+
     @Test
     public void invalidMacs() {
-      Arrays.asList(
-              MAGIC_MAC_START + "no spaces allowed",
-              MAGIC_MAC_START + "tooShort",
-              MAGIC_MAC_START + "#%@symbol",
-              Strings.repeat("0", ClientLoginValidator.MAC_ADDRESS_LENGTH)
-      ).forEach(invalidValue -> assertThat(ClientLoginValidator.isValidMac(invalidValue), is(false)));
+      final Collection<String> macs = Arrays.asList(
```

Comment:
Why the preference for the temporary variable? Excluding that strikes me as a more functional style, similar to the style of the `forEach` method.

Reply:
Not really my preference; just working around how the Eclipse Formatter works.  I preferred the style you used, but when you run the Formatter, it lines things up in such a way that makes it hard to distinguish the collection elements from the `.forEach` call.

I would love if Eclipse added an option to the Formatter to allow closing parentheses on a different line to permit the style you used, but I've yet to discover it. :smile: